app/views/course/announcements/_announcement.html.slim

```diff
@@ -0,0 +1,20 @@
+div class=("announcement #{ 'future' if announcement.valid_from > DateTime.now }")
+  h3
+    = announcement.title
+
+  - if can? :manage, Course::Announcement
+    .pull-right
```

Comment:
Actually I think we should follow slim's convention ? their sample code removes div everywhere( just like this).